client/lib/plugins/sanitize-section-content.js

```diff
@@ -132,6 +132,10 @@ const replacementFor = ( node ) => {
  * @returns {string} sanitized HTML
  */
 export const sanitizeSectionContent = ( content ) => {
+	// @TODO: replace with server-side sanitizer
+	if ( 'undefined' === typeof DOMParser ) {
```

Comment:
What are we planning to do here? I'd expect we handle sanitization somehow, otherwise bad bad things could happen?

Reply:
The plan is to follow up with the sanitizer changes after this PR.
Normally, this should never happen since the SSR routes are restricted behind the config flag. I've added another flag check so it's clearer and avoids any uncertainty. 

Once we have content sanitizing and react-query cache and hydration working, we should be safe to switch the flag.